### ISSUE TITLE
Progress Bar with Indeterminate Value not read correctly

### ISSUE THREAD

**Issue by jnurthen** [FreedomScientific/standards-support#353]
### Summary

The ARIA spec defines a progress bar with no aria-valuenow (and no aria-valuetext) as an Indeterminate progress bar. This should be read as such.

Go to https://codepen.io/jnurthen/full/qBErmdB
Note there are 2 progress bars on the page.

The first has a value (which is read)
The second does not have a value and is a loading progress indicator

### Expected result

JAWS should read "Loading, progress bar, indeterminate" (or another word - unknown value or such like)

### Actual result

On FF JAWS reads "50 Progress Bar partially checked"
On Chrome JAWS ignores the indeterminate Progress Bar completely

### Example

Go to https://codepen.io/jnurthen/full/qBErmdB
 2nd progress bar on the page

#### JAWS version and build number

JAWS 2018.1811.30

 
#### Operating System and version

Windows 10 19013.1122

#### Browser and version:

FF 71.0
Chrome 79.0.3945.88

**Comment by JAWS-test** [comment]
The output in IE 11 is like in Chrome: The progressbar is not output.

The problem persists with JAWS 2019

**Comment by kurtjwolf** [comment]
Similar progress bar issues filed with internal bug #113675

**Comment by stevefaulkner** [comment]
tested with JAWS 2024.2405.48 and Chrome Version 126.0.6478.127 (Official Build) (64-bit) announces as _indeterminate progress bar_
